### ISSUE TITLE
Fix user preferences in the /dashboard

### ISSUE THREAD

**Issue by iomodo** [oseducation/knowledge-graph#455]
If you go /dashboard and click on the preferences in the profile dropdown it does not work.

**Comment by meriam04** [comment]
Hello 👋

I am a Computer Engineering student and as part of my Software Engineering course, my team and I need to contribute to an open source project. My team and I were very interested in knowledge-graph and were hoping to help with some of the “good first issues”

So far, I have tried to follow the setup instructions outlined [here](https://github.com/oseducation/knowledge-graph/blob/main/server/README.md), and got a public API key for Youtube. However, when I try to use the `make import` command I get an error about a missing ChatGPT API key. 

I wanted to reach out over Discord, as suggested in the README, but that link to the server has expired. 

In summary, I would love to contribute to this project (if the team is willing) but for that to work I would need some instructions on the ChatGPT API key, and maybe the Discord server invite link could be updated too.

Thanks for your consideration!
Meriam 🙂

**Comment by iomodo** [comment]
Hey Meriam,
First of all, thank you for your interest in contributing to our project! We're thrilled to have enthusiastic students like you and your team onboard.

Regarding the ChatGPT API key issue, it's great that you've already set up the YouTube API key, that is all you need. For the ChatGPT API key we've updated the code and made it optional. If you want to chat with AI tutor you can set it up, but not it's necessary for this issue.

As for the Discord server link, we apologize for the inconvenience caused by the expired link. We have updated the README with a new, active Discord invite. Please check it out for joining our community where you can discuss and collaborate more efficiently.

@meriam04, please feel free to reach out for any further assistance on listed issues or features you'd like to work on. We're excited to see what you and your team will bring to the project!

Looking forward to your contributions!
Shota

**Comment by divyaK15** [comment]
Hello!

Could I be assigned this issue please?

Thanks,
Divya

**Comment by iomodo** [comment]
🚀
